string timestamp
jaco_manipulation/MoveItConfig config
jaco_manipulation/MoveItGoal goal
bool result